brecker.md

```diff
@@ -0,0 +1,7 @@
+Michael Brecker
+
+Early life and career
+Michael Brecker was born in Philadelphia and raised in Cheltenham Township, a local suburb. Born and raised in a Jewish family, his father Bob (Bobby) was a lawyer who played jazz piano and his mother Sylvia was a portrait artist.[4] Michael Brecker was exposed to jazz at an early age by his father. He grew up as part of the generation of jazz musicians who saw rock music not as the enemy but as a viable musical option. Brecker began studying clarinet at age 6, then moved to alto saxophone in eighth grade, settling on the tenor saxophone as his primary instrument in his sophomore year. He graduated from Cheltenham High School in 1967 and spent that summer at the Berklee College of Music in Boston. After a year at Indiana University he moved to New York City in 1969, where he carved out a niche for himself as a dynamic and exciting jazz soloist. He first made his mark at age 20 as a member of the jazz-rock band Dreams�a band that included his older brother, trumpeter Randy Brecker, trombonist Barry Rogers, drummer Billy Cobham, keyboardist Jeff Kent and bassist Doug Lubahn. Dreams was short-lived, lasting only from 1969 through 1972, but Miles Davis was seen at some gigs prior to his recording Jack Johnson.[5]
+
```

Comment:
You might want to mention that Michael recorded with his brother before joining Dreams.

Reply:
Okay, looks good.